src/components/Login.js

```diff
@@ -66,8 +66,8 @@ const Login = (props) => {
 
   const history = useHistory();
   const location = useLocation();
-  const { from } = location.state || { from: { pathname: '/' } };
-
+  const { from } = { from: { pathname: '/' } };
+  console.log(location);
```

Comment:
You can remove `console.log()` when development is complete so we don't pollute the console in production. If you need to keep logging in production, use `loglevel` instead.